crates/blockchain/Cargo.toml

```diff
@@ -13,6 +13,8 @@ ethereum_rust-storage.workspace = true
 ethereum_rust-evm.workspace = true
 sha3.workspace = true
 
+[dev-dependencies]
+serde_json = "1.0.117"
```

Comment:
This is already on the root cargo.toml, I think you should do instead: ```serde_json.workspace = true```

Reply:
Fixed.